index.html

```diff
@@ -8,7 +8,135 @@
     <title>Title</title>
 </head>
 <body>
-    <h1>Моя первая страница</h1>
-    <p><a href="./about/index.html"> Об авторе </a></p>
+<h1>Моя первая страница</h1>
+<p><a href="./about/index.html"> Об авторе </a></p>
+<hr>
+<p>На изучение Front-end меня натолкнули следующие причины: &#9878;</p>
+<p>
+    Выгорание на теперещней работе &#x1F525 <br>
+    Упёрся в потолок зарплаты &#x270A <br>
+    Желание учиться новому &#9872;<br>
+</p>
+<img src="./images/158px-Juventus_FC_2017_icon_(black).svg.png" alt="Footbal club juventus">
+<ol>
+    <li>Знакомство с Web</li>
+    <ul>
+        <li>Кратакая история Web и что такое интернет.</li>
+        <li>Как устроен браузер и как он получает страницы по сети.</li>
+        <li>Что такое Git?</li>
+        <li>Создание первого репозитория.</li>
+        <li>О файле .gitignore.</li>
+    </ul>
+    <li>HTML-вёрстка - первые шаги</li>
+    <ul>
+        <li>Настройка рабочей среды.</li>
+        <li>Краткая история HTML.</li>
+        <li>Какие бывают теги и что такое атрибут.</li>
+        <li>Теги для работы с текстом.</li>
+        <li>Создание собственной страницы.</li>
+    </ul>
+    <li>Погружение в HTML-разметку</li>
+    <ul>
+        <li>Мета-теги.</li>
+        <li>Мнемоники</li>
+        <li>Тег для работы с изображениями.</li>
+        <li>Теги для работы со списками.</li>
+        <li>Теги для работы с таблицами.</li>
+        <li>Теги для работы с формами.</li>
+        <li>DIV-блоки.</li>
+    </ul>
+</ol>
+<table border="1">
+    <thead>
+    <tr>
+        <th>Tag</th>
+        <th>Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td>&lt;table&gt;</td>
+        <td>Defines a table</td>
+    </tr>
+    <tr>
+        <td>&lt;tr&gt;</td>
+        <td>Defines a row in a table</td>
+    </tr>
+    <tr>
+        <td>&lt;th&gt;</td>
+        <td rowspan="2">Defines a "header cell" / "cell" in a table</td>
+    </tr>
+    <tr>
+        <td>&lt;td&gt;</td>
+    </tr>
+    <tr>
+        <td colspan="2">Rarely used</td>
+    </tr>
+    <tr>
+        <td>&lt;caption&gt;</td>
+        <td>Defines a table caption</td>
+    </tr>
+    <tr>
+        <td>&lt;thead&gt;</td>
+        <td>Groups the header content in a table</td>
+    </tr>
+    <tr>
+        <td>&lt;tbody&gt;</td>
+        <td>Groups the body content in a table</td>
+    </tr>
+    <tr>
+        <td>&lt;tfoot&gt;</td>
+        <td>Groups the footer content in a table</td>
+    </tr>
+    </tbody>
+</table>
+<p></p>
+<form action="/" method="get">
+    <fieldset style="width: 50%">
+        <legend>My first form</legend>
+        <div><label for="name">Name</label></div>
+        <div><input id="name" type="text" value="Name" required></div>
+        <div>Email</div>
+        <div><input type="email" required></div>
```

Comment:
add a name and id attribute that will allow you to create a url like in the example